integration_tests/models/schema_tests/schema.yml

```diff
@@ -13,7 +13,7 @@ models:
         tests:
           - dbt_utils.at_least_one
 
-  - name: data_test_expression_is_true
+  - name: data_test_expression_is_true_1
```

Comment:
Migrate old seed to new one

Reply:
Could you name the seeds with a bit more clarity? perhaps `data_test_expression_is_true` (i.e staying as-is) and `data_test_expression_is_true_window_functions` or something?

Reply:
Happy to. If this is something that you don't like, this pattern of seed (`*_1.csv`, `*_2.csv` etc...) is present in a few other test files FYI